### DISCUSSION TITLE
Keeping getting CLIENT_FETCH_ERROR after migrating to v4.3.3 (even after checking NEXTAUTH_URL AND NEXTAUTH_SECRET)

### DISCUSSION THREAD

**Discussion by y-arjun-y** [nextauthjs/next-auth#4470] [Help]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/58625220/165270114-3fd82cc7-b661-4f41-b8ab-ffa04bca4ffc.png">

On [this website](https://bettermailto.com), even after setting NEXTAUTH_URL and NEXTAUTH_SECRET in Vercel (and redeploying), I still get this error and "?error=Configuration" when clicking "Log In".

Any help would be appreciated!

**Comment by davidroulier** [comment]
Hi !

I see your website is working just fine now.

How did you fixed this error, please ?

Regards

**Reply by y-arjun-y** [reply]
I didn't fix the error, I just reverted to a previous version of next-auth.

**Reply by FemtDeveloper** [reply]
hi, which version did you let?

**Reply by y-arjun-y** [reply]
3.29.0

**Reply by Kipitup** [reply]
Any idea on why ?
